packages/flutter/test/widgets/scrollable_helpers_test.dart

```diff
@@ -670,4 +670,56 @@ void main() {
     },
     variant: KeySimulatorTransitModeVariant.all(),
   );
```

Comment:
Could you also add a test that verifies the reported issue with Column+Flexible works asa expected?